docs/data/sdks/typescript-react-native/index.md

```diff
@@ -650,4 +652,92 @@ amplitude.init(API_KEY, OPTIONAL_USER_ID, {
 });
 ```
 
+### Advertising Identifiers
+
+Different platforms have different advertising identifiers. Due to user privacy concerns, Amplitude does not automatically collect these identifiers. However, it is easy to enable them using the instructions below. It is important to note that some identifiers are no longer recommended for use by the platform providers. Please read the notes below before deciding to enable them.
+
+| Platform | Advertising Identifier | Recommended | Notes |
+| --- | --- |-------------| --- |
+| Android | AppSetId | Yes         | [AppSetId](https://developer.android.com/training/articles/app-set-id) is a unique identifier for the app instance. It is reset when the app is reinstalled. |
+| Android | ADID | No          | [ADID](https://support.google.com/googleplay/android-developer/answer/6048248?hl=en) is a unique identifier for the device. It is reset when the user opts out of personalized ads. |
+| iOS | IDFV | Yes         | [IDFV](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) is a unique identifier for the app instance. It is reset when the app is reinstalled. |
+| iOS | IDFA | No          | [IDFA](https://developer.apple.com/documentation/adsupport/asidentifiermanager/1614151-advertisingidentifier) is a unique identifier for the device. It is reset when the user opts out of personalized ads. |
+
+#### Android
+
+##### App set ID
+
+App set ID is a unique identifier for each app install on a device. App set ID is reset by the user manually when they uninstall the app, or after 13 months of not opening the app. Google designed this as a privacy-friendly alternative to Ad ID for users who want to opt out of stronger analytics.
+
+To use App Set ID, follow these steps.
+
+1. Add `play-services-appset` as a dependency to the Android project of your app.
+
+    ```bash
+    dependencies {
+        implementation 'com.google.android.gms:play-services-appset:16.0.2'
+    }
+    ```
+
+2. Enable `trackingOptions.appSetId`
+
+    ```ts
+    amplitude.init(API_KEY, OPTIONAL_USER_ID, {
+      trackingOptions: {
+        appSetId: true,
+      },
+    });
+    ```
+   
+##### Android Ad ID
```

Comment:
Should we add the same warning as idfa in iOS?

Reply:
I thought but `adid` is enabled by default, so it would be weird to call it out as "not recommended", so I just omitted mentioning it. 

Not 100% why it is enabled by default when others aren't. I feel it would make sense to disable by default, but it's a breaking change.

I'm going to omit the warning for now.